config/config.go

```diff
@@ -10,25 +10,30 @@ import (
 
 var DB_URL string
 var TASK_TIMEOUT int
-var SERVER_PORT string
+var SERVER_PORT int
 var IN_CLUSTER bool
 
 func InitConfig() {
+	var err error
 	initializers.LoadEnvVars()
 	DB_URL = os.Getenv("DB_URL")
-	parsedTimeout, err := strconv.Atoi(os.Getenv("TASK_TIMEOUT"))
 
+	TASK_TIMEOUT, err = strconv.Atoi(os.Getenv("TASK_TIMEOUT"))
 	if err != nil {
 		TASK_TIMEOUT = 300
-		log.Fatal("Can't parse tasktimeout, using default")
+		log.Println("Can't parse tasktimeout, using default")
```

Comment:
hmm why?

Reply:
Because fatal runs `os.exit` which means we will not be using the default